src/scripts/settings/appSettings.js

```diff
@@ -108,28 +107,40 @@ define(['appStorage', 'events'], function (appStorage, events) {
         }
 
         return [];
-    };
+    }
 
-    AppSettings.prototype.runAtStartup = function (val) {
+    export function runAtStartup(val) {
         if (val != null) {
             this.set('runatstartup', val.toString());
         }
 
         return this.get('runatstartup') === 'true';
-    };
+    }
 
-    AppSettings.prototype.set = function (name, value, userId) {
+    export function set(name, value, userId) {
         var currentValue = this.get(name, userId);
         appStorage.setItem(getKey(name, userId), value);
 
         if (currentValue !== value) {
             events.trigger(this, 'change', [name]);
         }
-    };
+    }
 
-    AppSettings.prototype.get = function (name, userId) {
+    export function get(name, userId) {
         return appStorage.getItem(getKey(name, userId));
-    };
+    }
 
-    return new AppSettings();
-});
+    export default {
+        enableAutoLogin: enableAutoLogin,
```

Comment:
Isn't the individual `export`s on the functions redundant since this does the same thing?

Reply:
I think @dmitrylyzo was adding them for compatibility reasons.

Reply:
Yes, this need for compatibility when you import ES6-rewritten module. Without `default`, I got error `default is undefined`, iirc
But I didn't try to remove individual exports leaving only default.

With individual and default exports, result will be:
```
{
    _esModule: true, <-- iirc
    func1: ...,
    func2: ...,
    default: {
        func1: ...,
        func2: ...
    }
}
```

Reply:
Ah alright, makes sense to me.